app/views/helpers/FullResultsPageHelper.scala

```diff
@@ -222,8 +274,8 @@ object FullResultsPageHelper extends ViewHelper {
         TableRow(content = Text(messages("fullResultsPage.financialYear.taxableProfitDistributions"))),
         TableRow(content =
           Text(
-            s"${CurrencyUtils.format(marginalRate.adjustedProfit)} + ${CurrencyUtils
-                .format(distributions)} × ($daysString ÷ $daysInYear $daysMsg)"
+            s"(${CurrencyUtils.format(taxableProfit)} + ${CurrencyUtils
+                .format(distributions)}) × ($daysString ÷ $daysInYear $daysMsg)"
```

Comment:
Shouldnt this be the length of the accounting period?

Reply:
Which variable are you talking about? daysInYear? 
In this case:
`val daysInYear = daysInFY(marginalRate.year)`

```
  def daysBetweenInclusive(start: LocalDate, end: LocalDate): Int =
    (start.until(end, ChronoUnit.DAYS) + 1).toInt

  def daysInFY(year: Int): Int = {
    val start = LocalDate.of(year, 4, 1)
    daysBetweenInclusive(start, start.plusYears(1).withMonth(3).withDayOfMonth(31))
  }
```

Is this wrong?

Reply:
For taxable profit and taxable profit with distributions, the ratio should be 

```ratio = number of accounting period days for the year/length of the accounting period)```